### ISSUE TITLE
[error] GenServer Indexer.Block.Realtime.Fetcher terminating

### ISSUE THREAD

**Issue by peachwaravich** [poanetwork/blockscout#2531]
I got a problem Error on blockscout docker 

`| 2019-08-09T03:08:20.496 fetcher=block_realtime [error] GenServer Indexer.Block.Realtime.Fetcher terminating
explorer_1  | ** (stop) exited in: GenServer.call(Indexer.Block.Realtime.WebSocket, {:subscribe, "newHeads", []}, 5000)
explorer_1  |     ** (EXIT) time out`

### Environment

* Elixir & Erlang/OTP versions (`elixir -version`): Elixir 1.8.0
* Operating System: Ubuntu 18.04 (Oracle Vm VirtualBox)

### Steps to reproduce
run docker-compose.yml 

```
version: '3'
services:
  postgres:
    build:
      context: .
      dockerfile: DockerfilePostgres
    restart: always
    expose:
      - "5432"
  explorer:
    build:
      context: .
      dockerfile: DockerfileExplorer
    restart: "no"
    tty: true
    environment:
      - DATABASE_URL=ecto://postgres:@postgres/explorer?ssl=false
      - ETHEREUM_JSONRPC_VARIANT=parity
      - ETHEREUM_JSONRPC_HTTP_URL=http://192.168.50.100:8541
      - ETHEREUM_JSONRPC_TRACE_URL=http://192.168.50.100:8541
      - ETHEREUM_JSONRPC_WS_URL=ws://192.168.50.100:8541
    volumes:
      - blockscout-data:/opt/app
    ports:
      - "4000:4000"
    depends_on:
      - postgres
volumes:
  blockscout-data:

```

run parity ethereum
```
version: '2.1'
services:
  validator2:
    build:
      context: .
      dockerfile: docker_authority/Dockerfile
      args:
        PARITY_VERSION: ${PARITY_VERSION}
        NETWORK_NAME: ${NETWORK_NAME}
        ID: 2
    volumes:
      - parity-data-2:/home/parity/.local/share/io.parity.ethereum/chains/${NETWORK_NAME}/db
      - ./parity/config:/parity/config
    command:
      --config /parity/config/authority.toml
      --engine-signer 0xa35F71ab0bfc20418392Cc7580######
      --unlock 0xa35F71ab0bfc20418392Cc7580#####
      --ws-apis all
      --ws-port 8541
      --jsonrpc-interface all
      --jsonrpc-cors all
      --no-persistent-txqueue
      --jsonrpc-hosts all
      --jsonrpc-server-threads 8
      --jsonrpc-threads 0
      --tracing on
      --pruning archive

    ports:
      - 8541:8545
      - 30303:30303
    networks:
      app_net:
        ipv4_address: 172.16.0.13

  monitor:
    image: buythewhale/ethstats_monitor
    volumes:
      - ./monitor/app.json:/home/ethnetintel/eth-net-intelligence-api/app.json:ro
  dashboard:
    image: buythewhale/ethstats
    volumes:
      - ./dashboard/ws_secret.json:/eth-netstats/ws_secret.json:ro
    ports:
      - 3001:3000
volumes:
  parity-data-2:


networks:
  app_net:
    driver: bridge
    ipam:
      driver: default
      config:
      - subnet: 172.16.0.1/24
        gateway: 172.16.0.1
```

### Actual behaviour

`| 2019-08-09T03:08:20.496 fetcher=block_realtime [error] GenServer Indexer.Block.Realtime.Fetcher terminating
explorer_1  | ** (stop) exited in: GenServer.call(Indexer.Block.Realtime.WebSocket, {:subscribe, "newHeads", []}, 5000)
explorer_1  |     ** (EXIT) time out`

**Comment by peachwaravich** [comment]
![image](https://user-images.githubusercontent.com/36033754/62753431-cb95db00-ba95-11e9-90f1-4d0eb2bc733a.png)

**Comment by vbaranov** [comment]
@peachwaravich which version of Blockscout do you use? The latest published release is compatible with Elixir 1.9.0

**Comment by vbaranov** [comment]
@peachwaravich  Aren't you under VPN? I can repeat this issue in local setup only if I am under VPN connection.

**Comment by peachwaravich** [comment]
> @peachwaravich which version of Blockscout do you use? The latest published release is compatible with Elixir 1.9.0

How to check version for Blockscout ?? , How can I do update Elixir ?
now I use config on puppeth/blockscout : 
https://github.com/puppeth/blockscout : use to Elixir 1.8.0

**Comment by peachwaravich** [comment]
thank you I solve a problem , because docker-compose not expose port ws
(web-socket)
